src/main/java/core/basesyntax/Circle.java

```diff
@@ -0,0 +1,21 @@
+package core.basesyntax;
+
+public class Circle extends Figure {
+    private final int radius;
+
+    public Circle(Color color,int radius) {
```

Comment:
There should be a space after the comma in the parameters list of the constructor.